packages/flutter_tools/lib/src/device.dart

```diff
@@ -559,7 +559,7 @@ abstract class Device {
         supportIndicator += ' ($type)';
       }
       table.add(<String>[
-        device.name,
```

Comment:
This will change everything, not just desktop; is that desired?

Reply:
That was *my* intent, yes.  Whether it's desirable it up to review. ;)